django/db/models/base.py

```diff
@@ -900,10 +900,12 @@ def save_base(
 
     save_base.alters_data = True
 
-    def _save_parents(self, cls, using, update_fields):
+    def _save_parents(self, cls, using, update_fields, updated_parents=None):
```

Comment:
Parents can be inserted or updated so I'd prefer naming proposed by Simon
```suggestion
    def _save_parents(self, cls, using, update_fields, saved_parents=None):
```

Reply:
Nevermind, in `updated_parents` we store whether parent was updated or not, so `updated_parents` is fine.